src/transformers/configuration_utils.py

```diff
@@ -159,6 +159,7 @@ def __init__(self, **kwargs):
         self.no_repeat_ngram_size = kwargs.pop("no_repeat_ngram_size", 0)
         self.bad_words_ids = kwargs.pop("bad_words_ids", None)
         self.num_return_sequences = kwargs.pop("num_return_sequences", 1)
+        self.chunk_size_feed_forward = kwargs.pop("chunk_size_feed_forward", 1)
```

Comment:
This should be disabled by default -> so would be nice to set it to `0`

Reply:
And to make sure we don't break backward compatibility...

Reply:
in modeling_bert.py, refer to #6972. thanks.